dynawo/sources/Solvers/Common/DYNSolverImpl.h

```diff
@@ -378,12 +389,15 @@ class Solver::Impl : public Solver, private boost::noncopyable {
   modeChangeType_t minimumModeChangeTypeForAlgebraicRestoration_;  ///< parameter to set the minimum mode level at which algebraic restoration will occur
   modeChangeType_t minimumModeChangeTypeForAlgebraicRestorationInit_;  ///< parameter to set the minimum mode level
                                                                        ///< at which algebraic restoration will occur at init
+  bool multipleStrategiesForAlgebraicRestoration_;  ///< parameter to activate multi strategy for algebraic restoration
+  bool printUnstableRoot_;  ///< to print unstable root in logs
 
   stat_t stats_;  ///< execution statistics of the solver
   double tSolve_;  ///< current internal time of the solver
   BitMask state_;  ///< current state value of the solver
 
   bool startFromDump_;  ///< is solver starting from dump
+  int numDifferentialVariables_;  ///< number of differential variables
```

Comment:
This belongs to the model world, why storing it in the solver? Moreover this can change during the simulation depending on discrete variables values

Reply:
That's not a debugging log it's what the user wants to see, the number of variables and the number of differential variables to have an idea of the size of the problem.